### PR TITLE
fix(step-description): duplicated titles in proposal with edit type (#1735)

### LINKED ISSUE #1735: Titles are duplicated when extending role assignments
Reproduce:
1. Extend a role assignment
2. Check title

Expected behaviour:
Title should be "Strategic Anchor Council"

Experienced behaviour:
Title is duplicated: "Strategic Anchor Council: Strategic Anchor Council"

e.g. https://dao.hypha.earth/hypha/proposals/37101
e.g. https://dao.hypha.earth/hypha/proposals/

image

image

### PR DESCRIPTION
**Pull request by evgenibir**
### 🗃 Github Issue Or Explanation for this PR. (What is it supposed to do and Why is needed)

Titles are duplicated when extending role assignments #1735

### ✅ Checklist

- Fixed duplicate titles in proposal with edit type

close #1735

### CODE REVIEW

File: src/pages/proposals/create/StepDescription.vue [hunk 1]
```diff
@@ -136,6 +142,7 @@ widget(:class="{ 'disabled': currentStepName !== 'step-description' && $q.screen
         dense
         outlined
         v-model="title"
+        :disable="this.$store.state.proposals.draft.edit"
```
Comment (arsenijesavic) [inline src/pages/proposals/create/StepDescription.vue:145]: @evgenibir you dont need `this.` when you use in component props

File: src/pages/proposals/create/StepDescription.vue [hunk 2]
```diff
@@ -35,7 +35,14 @@ export default {
 
   computed: {
     nextDisabled () {
-      if (this.title.length > 0 && this.$sanitize(this.description, { allowedTags: [] }).length < DESCRIPTION_MAX_LENGTH && this.title.length <= TITLE_MAX_LENGTH) {
+      if (this.$store.state.proposals.draft.edit) {
+        if (this.$sanitize(this.description, { allowedTags: [] }).length < DESCRIPTION_MAX_LENGTH) {
```
Comment (arsenijesavic) [inline src/pages/proposals/create/StepDescription.vue:39]: can you compute this value to something like `sanitizedDescription` so you can reuse it and it removes clutter?

Comment (arsenijesavic) [reply src/pages/proposals/create/StepDescription.vue:39]: @evgenibir

**Review by arsenijesavic — CHANGES_REQUESTED** [review]

File: src/pages/proposals/create/StepDescription.vue (hunk 2 above)
Comment (evgenibir) [reply src/pages/proposals/create/StepDescription.vue]: @arsenijesavic ready!

**Review by arsenijesavic — APPROVED** [review]

**Pull request merged**